src/strategies/masterchef-pool-balance-price/index.ts

```diff
@@ -0,0 +1,450 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+import { BigNumber} from '@ethersproject/bignumber';
+import fetch from 'cross-fetch';
+import { inspect } from 'util';
```

Comment:
@joaomajesus Can you just remove `inspect`? This wont work on browser

Reply:
ah! ok will do it right away

Reply:
done